2025-06-11 23:15:39,698 ; pylayers.util.project ; INFO ; INFO
2025-06-11 23:15:39,698 ; pylayers.util.project ; DEBUG ; DEBUG
2025-06-11 23:15:39,699 ; pylayers.util.project ; ERROR ; ERROR
2025-06-11 23:15:39,699 ; pylayers.util.project ; CRITICAL ; CRITICAL
2025-06-11 23:15:39,699 ; pylayers.util.project ; WARNING ; WARNING
2025-06-11 23:15:40,354 ; pyface.base_toolkit ; INFO ; Could not load traitsui.toolkits plugin 'qt' from 'traitsui.qt'
2025-06-11 23:15:40,354 ; pyface.base_toolkit ; DEBUG ; Cannot import any of PySide2, PySide6, PyQt5, PyQt6
Traceback (most recent call last):
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/pyface/base_toolkit.py", line 288, in find_toolkit
    toolkit = plugin.load()
              ^^^^^^^^^^^^^
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/importlib/metadata/__init__.py", line 202, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/traitsui/qt/__init__.py", line 29, in <module>
    import pyface.qt
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/pyface/qt/__init__.py", line 47, in <module>
    raise ImportError("Cannot import any of " + ", ".join(modules))
ImportError: Cannot import any of PySide2, PySide6, PyQt5, PyQt6
2025-06-11 23:15:40,356 ; pyface.base_toolkit ; INFO ; Could not load traitsui.toolkits plugin 'wx' from 'traitsui.wx'
2025-06-11 23:15:40,356 ; pyface.base_toolkit ; DEBUG ; No module named 'wx'
Traceback (most recent call last):
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/pyface/base_toolkit.py", line 288, in find_toolkit
    toolkit = plugin.load()
              ^^^^^^^^^^^^^
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/importlib/metadata/__init__.py", line 202, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/traitsui/wx/__init__.py", line 19, in <module>
    from . import toolkit
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/traitsui/wx/toolkit.py", line 22, in <module>
    import wx
ModuleNotFoundError: No module named 'wx'
2025-06-11 23:15:40,358 ; pyface.base_toolkit ; INFO ; Could not load traitsui.toolkits plugin 'qt4' from 'traitsui.qt'
2025-06-11 23:15:40,358 ; pyface.base_toolkit ; DEBUG ; Cannot import any of PySide2, PySide6, PyQt5, PyQt6
Traceback (most recent call last):
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/pyface/base_toolkit.py", line 288, in find_toolkit
    toolkit = plugin.load()
              ^^^^^^^^^^^^^
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/importlib/metadata/__init__.py", line 202, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/traitsui/qt/__init__.py", line 29, in <module>
    import pyface.qt
  File "/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/pyface/qt/__init__.py", line 47, in <module>
    raise ImportError("Cannot import any of " + ", ".join(modules))
ImportError: Cannot import any of PySide2, PySide6, PyQt5, PyQt6
2025-06-11 23:15:40,416 ; apptools.preferences.preferences ; DEBUG ; loading preferences from </Users/nisargpatel/.enthought/mayavi_e3/preferences.ini>
2025-06-11 23:15:40,420 ; apptools.preferences.preferences ; DEBUG ; loading preferences from <<_io.BufferedReader name='/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/mayavi/preferences/preferences.ini'>>
2025-06-11 23:15:40,421 ; apptools.preferences.preferences ; DEBUG ; loading preferences from <<_io.BufferedReader name='/Users/nisargpatel/miniforge3/envs/pylayers-py3/lib/python3.11/site-packages/tvtk/plugins/scene/preferences.ini'>>
